cmd/server/assets/realmadmin/_stats_keyserver.html

```diff
@@ -0,0 +1,118 @@
+{{define "realmadmin/_stats_keyserver"}}
+
+<div class="card shadow-sm mb-3">
+  <div class="card-header">
+    <span class="oi oi-bar-chart mr-2 ml-n1"></span>
+    Key server stats
+  </div>
+  <div id="keyserver_chart_div" class="h-100 w-100" style="min-height:325px;">
+    <p class="text-center font-italic w-100 mt-5">Loading chart...</p>
+  </div>
+  <small class="card-footer d-flex justify-content-between text-muted">
+    <a href="#" data-toggle="modal" data-target="#keyserver-stats-modal">Learn more about this chart</a>
+    <span>
+      <span class="mr-1">Export as:</span>
+      <a href="/stats/realm/key-server.json">JSON</a>
```

Comment:
No CSV?

Reply:
The distribution graphs (that I haven't added yet) have another array data dimension that's hard to format as CSV. Haven't done a csv marshaller yet